lib/tidewave/mcp/tools/source.ex

```diff
@@ -26,6 +28,33 @@ defmodule Tidewave.MCP.Tools.Source do
           }
         },
         callback: &get_source_location/1
+      },
+      %{
+        name: "get_package_location",
+        description: """
+        Returns the location of dependency packages.
+
+        You can use this tool to get the location of any project dependency. Optionally,
+        a specific dependency name can be provided to only return the location of that dependency.
+
+        Use the result in combination with shell tools like grep to look for specific
+        code inside dependencies.
+
+        Do NOT use `list_project_files` to inspect dependencies. It will NOT return files from dependencies,
+        but only files from the current project itself.
```

Comment:
```suggestion
        code inside dependencies.
```

I don't think we should mention `list_project_files` but perhaps we should add a note to the response, rather than the tool description, saying that dependencies that have an absolute path belong outside the project root and may only be readable by shell commands?